pkg/identity/undeploy.go

```diff
@@ -23,6 +23,12 @@ func Undeploy(ctx context.Context, log *logger.Logger, clientset kubernetes.Inte
 	if err := deleteSecret(ctx, clientset, namespace); err != nil {
 		return errors.Wrap(err, "failed to delete secret")
 	}
+	if err := deletePostgresSecret(ctx, clientset, namespace); err != nil {
```

Comment:
this does not delete the db does it? if it does not are you sure you want to delete this secret and job?

Reply:
hmmm good point, it does not